src/cache/cache_las.c

```diff
@@ -1270,8 +1283,17 @@ __wt_las_sweep(WT_SESSION_IMPL *session)
             WT_ERR(cursor->next(cursor));
         }
 
-        /* Don't remove the non obsolete entry. */
-        if (las_timestamp > S2C(session)->txn_global.oldest_timestamp ||
+        /*
+         * There are several conditions that need to be met before we choose to remove a lookaside
+         * entry:
+         *  * If there exists a last checkpoint timestamp, then the lookaside record timestamp must
```

Comment:
Nit: Maybe use a hyphen instead of asterisk since we're already using an asterisk at the beginning of each line in a comment.

Reply:
Added numbers